frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/moderation/ModerationPortlet.java

```diff
@@ -112,27 +112,37 @@ private void renderActionView(RenderRequest request, RenderResponse response) th
                 ModerationService.Iface client = thriftClients.makeModerationClient();
                 ModerationRequest moderationRequest = client.getModerationRequestById(id);
                 String action = request.getParameter(ACTION);
+                String moderationCommentDecoded = request.getParameter(MODERATION_DECISION_COMMENT);
```

Comment:
given that you decode the contents of this variable in line 118, it follows that it should be called `moderationCommentEncoded`